### ISSUE TITLE
Missing Center-Alignment for Headings and Images with Gutenberg

### ISSUE THREAD

**Issue by ianstewart** [Automattic/themes#1856]
A few of our free themes are missing center-alignment for headings and images.

**Twenty Nineteen** has offset headings when they're center-aligned.

**Sketch** does not correctly center-align images. Neither does **Franklin**.

![image](https://user-images.githubusercontent.com/545779/77098740-d415a300-69e0-11ea-98d3-db2c23e170fd.png)

Related:

https://github.com/Automattic/wp-calypso/issues/28756

cc @lancewillett

**Comment by lancewillett** [comment]
Thanks for tracking it down, @ianstewart

**Comment by kjellr** [comment]
Twenty Nineteen needs a patch in core to fix those headers. I've created one here: https://core.trac.wordpress.org/ticket/49699

As for Sketch and Franklin, I've tested these on my end and I'm only able to consistently recreate in the editor, and only when I follow the following steps: 

1. Add an image.
2. Resize it to be a custom size using the handles or the sidebar.
3. Set the image to be center-aligned.

This appears to be a general Gutenberg bug, since I see it in all themes. I've filed a bug here: https://github.com/WordPress/gutenberg/issues/21152

Otherwise, images appear to be center-aligned just fine: 

![Screen Shot 2020-03-25 at 11 55 07 AM](https://user-images.githubusercontent.com/1202812/77559624-c6559700-6e92-11ea-87af-7dbf4cfc906d.png)

Is there something I should do differently to reproduce?
